### DISCUSSION TITLE
INAV 3.0.1 update says to restore with dump, instead of diff

### DISCUSSION THREAD

**Discussion by TTGG4400** [iNavFlight/inav#7267] [Q&A] [answered]
The update instructions for INAV 3.0.1, when updating from INAV 3.0.0, say to 1) take a dump, then 2) load 3.0.1 using full chip erase, and then 3) restore settings by loading the dump.  

Is this for real?  Or did they mean to say to use diff all, rather than dump?

**Comment by OptimusTi** [comment]
dump and dump all   are valid CLI commands.

You can find the CLI Commands reference here:  https://github.com/iNavFlight/inav/blob/master/docs/Cli.md

**Comment by TTGG4400** [comment]
Thx, but I have used the CLI quite a bit.  What I was asking about was the use of dump, instead of diff, for restoring setup after a FW update.  All the FW updates I've done in the past - used diff for restoring setup after the update - not dump.  And when I say used diff, I mean before the FW update you go into the CLI and do a diff all and save that.  Then after the FW update, you take that saved file, and put it back into the CLI.

**Comment by TTGG4400** [comment, marked as answer]
### DONE I got tired of waiting for clarification - and I assumed the docs were in error - so I took a diff all of 3.0.0, and updated to 3.0.1 using full chip erase, and I restored my setup using the INC presets along with the diff all I'd taken beforehand.  This worked, BTW.  The docs are wrong about using a dump to restore setup.

**Reply by DzikuVx** [reply]
The docs are not wrong. Both dump and diff will give positive results

**Reply by TTGG4400** [reply]
Thx
